src/Compilers/Core/Portable/PublicAPI.Shipped.txt

```diff
@@ -1634,6 +1634,7 @@ Microsoft.CodeAnalysis.SyntaxNodeOrTokenList.ToFullString() -> string
 Microsoft.CodeAnalysis.SyntaxReference
 Microsoft.CodeAnalysis.SyntaxReference.SyntaxReference() -> void
 Microsoft.CodeAnalysis.SyntaxRemoveOptions
+Microsoft.CodeAnalysis.SyntaxRemoveOptions.UseOriginalEndOfLine = 64 -> Microsoft.CodeAnalysis.SyntaxRemoveOptions
 Microsoft.CodeAnalysis.SyntaxRemoveOptions.AddElasticMarker = 32 -> Microsoft.CodeAnalysis.SyntaxRemoveOptions
```

Comment:
When we make API changes, we make them to the Unshipped file (since they are not shipped yet)

Reply:
Corrected